libs/ui/select/helm/src/lib/hlm-select-trigger.component.ts

```diff
@@ -46,11 +47,9 @@ type SelectTriggerVariants = VariantProps<typeof selectTriggerVariants>;
 	`,
 })
 export class HlmSelectTriggerComponent {
-	@ViewChild('button', { static: true })
-	public buttonEl!: ElementRef;
+	public buttonEl = viewChild<ElementRef>('button');
 
-	@ContentChild(HlmIconComponent, { static: false })
-	protected icon!: HlmIconComponent;
+	protected icon = contentChild(HlmIconComponent);
```

Comment:
same with this one

Reply:
there is no clear line in our repo right now. 
currently its about 50/50 if contentChild are readonly or not.
I would say we make it readonly.